ot/helpers/openmp_helpers.py

```diff
@@ -9,7 +9,12 @@
 import textwrap
 import subprocess
 
-from distutils.errors import CompileError, LinkError
+try:
+    from setuptools.errors import CompileError, LinkError
+except ImportError:
+    # Can remove once POT is Python 3.10+
+    # c.f. https://peps.python.org/pep-0632/
+    from distutils.errors import CompileError, LinkError
```

Comment:
I should think on a better way to just remove `distutils`. Will do a check, but maybe just add `setuptools` to `install_requires`.